src/ui/components/ProtocolViewer.tsx

```diff
@@ -123,38 +131,54 @@ const ProtocolViewer = () => {
   const selectedRequestDetails = useSelector(getFullRequestDetails(selectedRequests));
 
   return (
-    <div className="max-h-full overflow-y-scroll p-4">
-      {selectedRequestDetails.map(({ request, response, error }) => {
-        return (
-          <ProtocolRequestDetail
-            key={request!.id}
-            request={request!}
-            response={response}
-            error={error}
-          />
-        );
-      })}
-      <h3 className="text-lg">Protocol Info</h3>
-      {chunks.map(chunk => {
-        return (
-          <div
-            key={`${chunk.method}:${chunk.startedAt}`}
-            className={classNames("flex justify-between p-1", {
-              "text-lightGrey": chunk.pending,
-              "text-errorColor": chunk.errored,
-            })}
-            onClick={() => {
-              setSelectedRequests(chunk.ids);
-            }}
-          >
-            <span>
-              {chunk.method}
-              {chunk.count > 1 ? `(${chunk.count})` : null}
-            </span>
-            <span>{msAsMinutes(chunk.startedAt)}</span>
-          </div>
-        );
-      })}
+    <div className={styles.Container}>
```

Comment:
My only comment is: this could be a good place for an accordion 🤷‍♂️

Reply:
Oh! That is a fantastic idea.

Reply:
We seem to have at least **four** accordion components in the devtools repo 😁  

The first two I've looked at don't support dynamic children (e.g. rendering with only the requests pane at first and adding details panes later). They will throw if you do that. So they wouldn't work very well for this case.

I think I'll leave that out for now 😁